brainscore/metrics/IndividualCeiling.py

```diff
@@ -0,0 +1,56 @@
+from brainscore.metrics.ceiling import Ceiling
+from brainscore.metrics.transformations import Transformation
+from brainscore.benchmarks._neural_common import average_repetition
+import numpy as np
+import xarray as xr
+
+
+class InterIndividualCrossValidation:  # TODO inheritance Transformation???: InterIndivCrossVal = pipe, Ceiling = apply?

```

Comment:
I was not sure how inheritance in this case works because I could make this work with your `Transformation `class, at least I think, but after giving it some thought I found it a little counter-intuitive as `InterIndividualBenchmarkCeiling` should be a `Ceiling`.

Reply:
I think you could directly use the `CrossValidation` class inside of the ceiling class. If you set the type to kfold and the number of splits to equal to the number of subjects, it should always split the assembly into {pool, heldout} per split. https://github.com/mschrimpf/brain-score/blob/ebd6af2ce73e9837665a9b874e423ff875470683/brainscore/metrics/behavior_differences.py#L92-L93 is similar to that, holding out one task per split (instead of subject)

Reply:
I think the problem is that the statistics are not assemblies anymore but np.arrays. Do you think it is worth converting?

Reply:
Up to you -- I have kept things in assembly format just because I find it easier to work with labeled data, but you're building the benchmark, so you can decide what level of metadata you're comfortable with